examples/handlebars_templates/src/main.rs

```diff
@@ -15,7 +15,9 @@ use handlebars::{Helper, Handlebars, RenderContext, RenderError, JsonRender};
 #[derive(Serialize)]
 struct TemplateContext {
     name: String,
-    items: Vec<String>
+    items: Vec<String>,
+    title: String,
+    parent: String,
```

Comment:
Can we make this `&'static str`? My guess is that's what you'd want in a real application anyway.